cfgov/unprocessed/root/robots.txt

```diff
@@ -12,5 +12,13 @@ Disallow: /ask-cfpb/search/
 Disallow: /ask-cfpb/search-by-tag/
 Disallow: /es/obtener-respuestas/buscar/
 Disallow: /es/obtener-respuestas/buscar-por-etiqueta/
+Disallow: *&authors
+Disallow: *?authors
+Disallow: *&categories
+Disallow: *?categories
+Disallow: *&from_date
+Disallow: *&to_date
+Disallow: *?from_date
+Disallow: *?to_date
```

Comment:
I can never totally wrap my head around how robots.txt directives are parsed, but I *think* we can condense these and get the same result of disallowing URLs with the filter parameters in them:

```suggestion
Disallow: /*authors=
Disallow: /*categories=
Disallow: /*from_date=
Disallow: /*to_date=
Disallow: /*page=
```

I also added `Disallow: /*page=` in there. We have `Disallow: /?page=` on line 8 right now, but that won't catch URLs where the `page` parameter happens to not be the first parameter (like /?foo=bar&page=3), right?

I guess we can confirm that works in [Google's robots.txt tester](https://support.google.com/webmasters/answer/6062598), but that requires a Google account (I can log in on my personal machine and try it if that's easiest).